### ISSUE TITLE
Question: Time series simulation with scenarios

### ISSUE THREAD

**Issue by buddih09** [e2nIEE/simbench#26]
I saw the tutorial example where one can run a time series simulation with pre-defined load profiles. Similarly, is there a way to run a time series simulation for a pre-defined scenario (for example high load)?

**Comment by SteffenMeinecke** [comment]
Yes, that is also exemplified by the chapter _Run and Analyze Predefined Study Cases_ in the tutorial.
